packages/devtools_app/lib/src/screens/inspector/inspector_screen.dart

```diff
@@ -565,6 +569,63 @@ class _ErrorNavigatorButton extends StatelessWidget {
   }
 }
 
+class InspectorDetailsOption extends StatelessWidget {
+  const InspectorDetailsOption({super.key});
+
+  @override
+  Widget build(BuildContext context) {
+    return ValueListenableBuilder(
+      valueListenable: preferences.inspector.defaultInspectorDetailsView,
```

Comment:
Whatever the class gets renamed to, lets have the value listenable match (e.g. `preferences.inspector.defaultTab`)